src/info.c

```diff
@@ -1373,10 +1373,20 @@ ABTU_ret_err static int print_all_thread_stacks(ABTI_global *p_global, FILE *fp)
     size_t i;
     int abt_errno;
     struct info_pool_set_t pool_set;
+    struct tm *tm = NULL;
+    time_t seconds;
 
     abt_errno = info_initialize_pool_set(&pool_set);
     ABTI_CHECK_ERROR(abt_errno);
     ABTI_xstream *p_xstream = p_global->p_xstream_head;
+
+    seconds = (time_t)ABT_get_wtime();
+    tm = localtime(&seconds);
+    ABTI_ASSERT(tm != NULL);
+    fprintf(fp, "Start of ULT stacks dump %04d/%02d/%02d-%02d:%02d:%02d\n",
+	    tm->tm_year + 1900, tm->tm_mon + 1, tm->tm_mday, tm->tm_hour,
```

Comment:
Please replace tabs with whitespaces here.

The following script can automatically fix a whitespace issue.
```sh
bash maint/code-cleanup.sh src/info.c
```